posts/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+# Create your models here.
+class Post(models.Model):
+    title = models.CharField(max_length=20)
+    content = models.TextField()
```

Comment:
Zastanów się jak można zlimitować także ilość znaków (np do 1024) dla TextField, sprawdź także jaką maksymalną ilość znaków można zapisać defaultowo w TextField i jakie może rodzić to problemy

https://docs.djangoproject.com/en/4.1/ref/models/fields/#textfield
https://docs.djangoproject.com/en/3.2/ref/models/instances/#validating-objects

Reply:
manage.py shell_plus

Reply:
nadpisanie save()